lib/Doctrine/Common/PropertyChangedListener.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Doctrine\Common;
+
+/**
+ * Contract for classes that are potential listeners of a <tt>NotifyPropertyChanged</tt>
```

Comment:
Please use `{@see MyClass}`

Reply:
The cyclical dependency of these classes is a bit hard to grasp: can we add another sentence or two that don't involve explaining this with `NotifyPropertyChanged` in mind?